jetbrains-core/src/software/aws/toolkits/jetbrains/services/cloudwatch/logs/LogActor.kt

```diff
@@ -46,7 +46,7 @@ sealed class LogActor<T>(
         LOG.error(e) { "Exception thrown in the LogStreamActor not handled:" }
         notifyError(title = message("general.unknown_error"), project = project)
         table.setPaintBusy(false)
-        Disposer.dispose(this)
```

Comment:
Is this class registered to a parent Disposable?

Reply:
Yes, but, one of them was not registered, fixed the memory leak :D